index.html

```diff
@@ -185,5 +185,25 @@ <h2 class="third-text"><a>let's connect</a></h2>
           </li>
         </ul>
     </section>
+    <section class="contact-form">
+        <ul>
+            <li><img id="ellipse-12-2" src="elements/ellipse-12-2.png" alt="icon-7"></li>
+            <li><img id="ellipse-5-w" src="elements/ellipse-5-white.png" alt="icon-8"></li>
+            <li><img id="ellipse-3-w" src="elements/ellipse-3-white.png" alt="icon-9"></li>
+            <li><img id="rect-4-d" src="elements/rectangle-4-dark.png" alt="icon-10"></li>
+            <li><img id="rect-7-2" src="elements/rectangle-7-2.png" alt="icon-11"></li>
```

Comment:
- [ ] Make sure to use these images as background images and not `img` tags. It will get distorted and come over the Textarea field, so it's better to make all this as background images to fix the issue.

Reply:
hey, thanks for the feedback  i really appreciate, would you teach me how to make those elements background-images ?